subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/GradleBuildInstantExecutionSmokeTest.groovy

```diff
@@ -44,14 +44,9 @@ class GradleBuildInstantExecutionSmokeTest extends AbstractSmokeTest {
     }
 
     def "can build gradle with instant execution enabled"() {
-
         given:
         new TestFile("build/gradleBuildCurrent").copyTo(testProjectDir.root)
 
-        and:
-        def buildJavaHome = AvailableJavaHomes.getAvailableJdks(new GradleBuildJvmSpec()).last().javaHome
-        file("gradle.properties") << "\norg.gradle.java.home=${buildJavaHome}\n"
```

Comment:
:x: this should be reinstated, see https://github.com/gradle/gradle/pull/12625/files#r397659151